config/default.json

```diff
@@ -0,0 +1,32 @@
+{
+    "environment": "production",
+    "cookies": {
+        "token": "__auth__",
+        "userId": "__userid__"
+    },
+    "now": false,
+    "services": {
+        "idam": {
+            "idamApiUrl": "https://idam-api.platform.hmcts.net",
+            "idamClientID": "xuiwebapp",
```

Comment:
cant use xuiwebapp as a client id as that is for ExUi

Reply:
Changed